data/STRchive-loci.json

```diff
@@ -2560,13 +2560,13 @@
   "additional_literature": ["pmid:39576755", "pmid:39314239", "pmid:38605207", "pmid:37547453", "pmid:37456840", "pmid:37316299", "pmid:35982790", "pmid:35497811", "pmid:34641504", "pmid:34452200", "pmid:33672244", "pmid:33001366", "pmid:32451462", "pmid:32293552", "pmid:31213370", "pmid:30593830", "pmid:29520014", "pmid:29328069", "pmid:29217307", "pmid:29156055", "pmid:29052568", "pmid:28581490", "pmid:28407289", "pmid:27957769", "pmid:27036738", "pmid:27367740", "pmid:27340743", "pmid:27157321", "pmid:26943180", "pmid:26838233", "pmid:26693201", "pmid:26692014", "pmid:26498650", "pmid:25753030", "pmid:24718884", "pmid:24509297", "pmid:24416403", "pmid:24246952", "pmid:24233342", "pmid:23778023", "pmid:23770070", "pmid:23652307", "pmid:23317217", "pmid:23259747", "pmid:22970023", "pmid:21998660", "pmid:21385452", "pmid:20876819", "pmid:20562098", "pmid:20470225", "pmid:21637607", "pmid:19811637", "pmid:19625949", "pmid:19534821", "pmid:18619437", "pmid:18094420", "pmid:18021186", "pmid:17974963", "pmid:17694298", "pmid:17581677", "pmid:17203187", "pmid:17050588", "pmid:16711252", "pmid:16631167", "pmid:16302687", "pmid:16101182", "pmid:15991935", "pmid:15944787", "pmid:15814824", "pmid:15729696", "pmid:15604091", "pmid:15387710", "pmid:15115750", "pmid:15041735", "pmid:14871854", "pmid:14707484", "pmid:12747745", "pmid:12646731", "pmid:12626424", "pmid:12090474", "pmid:11923240", "pmid:11445551", "pmid:11391628", "pmid:11358826", "pmid:11295060", "pmid:11169964", "pmid:10797294", "pmid:10741704", "pmid:10653872", "pmid:10652432", "pmid:10541345", "pmid:10430099", "pmid:10389761", "pmid:10383817", "pmid:10235488", "pmid:10206297", "pmid:10052816", "pmid:10024673", "pmid:10022471", "pmid:9935162", "pmid:9823312", "pmid:9755875", "pmid:9727561", "pmid:9690452", "pmid:9591045", "pmid:9579805", "pmid:9575675", "pmid:9551361", "pmid:9427605", "pmid:9419405", "pmid:8967520", "pmid:8643693", "pmid:7594478", "pmid:8579785", "pmid:8567787", "pmid:8519447", "pmid:7628867", "pmid:7816840", "pmid:7946402", "pmid:7514493", "pmid:7690213", "pmid:7685318"]
 },
 {
-  "chrom": "chr15",
+  "chrom": "chr1",
   "start_hg38": 156591766,
-  "stop_hg38": 156591783,
+  "stop_hg38": 156591780,
   "start_hg19": 156561558,
-  "stop_hg19": 156561575,
+  "stop_hg19": 156561572,
   "start_t2t": 155728132,
-  "stop_t2t": 155728149,
+  "stop_t2t": 155728146,
```

Comment:
I think this should be:
"stop_t2t": 155728157,
It looks to me like there are 5 motifs in T2T (while there are 3 in hg38 and hg19)